substrate/pkg/controller/substrate/infrastructure/routetable.go

```diff
@@ -87,10 +88,10 @@ func (r *routeTable) Delete(ctx context.Context, substrate *v1alpha1.Substrate)
 	return reconcile.Result{}, nil
 }
 
-func privateTableName(identifier string) string {
-	return fmt.Sprintf("%s-%s", identifier, "private")
+func privateTableName(substrate *v1alpha1.Substrate) string {
+	return fmt.Sprintf("%s-%s", substrate.Name, "private")
 }
```

Comment:
Should we be using the same pattern here as in subnets?
```suggestion
func routeTableName(substrate *v1alpha1.Substrate, public bool) *string {
	if public {
		return discovery.Name(substrate, "public")
	}
	return discovery.Name(substrate, "private")
}
```